flink-metrics/flink-metrics-jmx/src/test/java/org/apache/flink/metrics/jmx/JMXReporterTest.java

```diff
@@ -130,10 +130,10 @@ public Integer getValue() {
 		};
 
 		rep1.notifyOfAddedMetric(g1, "rep1", new FrontMetricGroup<>(
-			new ReporterScopedSettings(0),
+			new ReporterScopedSettings(0, '.'),
```

Comment:
we could have a testing factory method for the `ReporterScopedSettings`. Then we have a single place where the `ReporterScopedSettings` are being created. In the future, if we add more fields to it, then we have a single place where to add the default values to.